docs/Features-overview.md

```diff
@@ -89,11 +89,11 @@ ANSI terminals support a variety of features by including ANSI escape sequences
 
 The following are examples of output rendered by the same view code in these three different contexts.
 
-In PowerShell on Windows with ANSI mode enabled:
+In PowerShell on Windows with VT mode enabled:
```

Comment:
What is VT? Consider expanding the acronym the first time you refer to "VT". Also, consider adding a link to something that may help introduce VT to CLI newbies :)